### PR TITLE
[DA-137] Wrong convert of community versions to OSGi one

### PR DESCRIPTION
**Pull request by jomrazek**

### CODE REVIEW

File: common/src/main/java/org/jboss/da/common/version/OSGiVersionParser.java [hunk 1]
```diff
@@ -8,8 +8,13 @@
 public class OSGiVersionParser {
 
     public String getOSGiVersion(String version) {
-        Version osgi = new Version(version);
-        return osgi.getOSGiVersionString();
+        if (version.matches(".*[.-]redhat-(\\d+)\\D*.*"))
```
Comment (jbartece) [inline common/src/main/java/org/jboss/da/common/version/OSGiVersionParser.java]: Add javadoc please

Comment (jbartece) [inline common/src/main/java/org/jboss/da/common/version/OSGiVersionParser.java]: Probably the \\D*.* is should not be there.

**Comment by janinko** [comment]
:+1:

**Comment by thescouser89** [comment]
:+1:

**Pull request merged**